frontend/javascripts/admin/admin_rest_api.ts

```diff
@@ -2053,6 +2053,8 @@ export async function getPricingPlanStatus(): Promise<APIPricingPlanStatus> {
   return Request.receiveJSON("/api/pricing/status");
 }
 
+export const cachedGetPricingPlanStatus = _.memoize(getPricingPlanStatus);
```

Comment:
Caching is not without its own problems. The status might update (e.g. new people will join, more datasets will be uploaded taking up space etc...) 
How does this cache live? Until the next page refresh, right? So, the above points might not be a such a big issue after all. 

The approach is fine by me.

Reply:
> Until the next page refresh, right?

Yes, exactly. When visiting the orga page, the non-cached version is used. So, only the dashboard is affected. If outdated data is noticed, the user will likely hit refresh to fix it.